conf/messages

```diff
@@ -1289,29 +1301,37 @@ create.success.personId=VOA Personal ID:
 create-account.type.question=Which type of account would you like to create?
 
 enrolment.title = Complete your contact details
-enrolment.organisation.name = Business name
+enrolment.organisation.name = <h1 class="govuk-heading-m">Business name</h1>
 enrolment.organisation.name.label = Organisation name
 
 enrolment.assistant.hint = You have been added as a user to your organisation, please confirm your details below
 enrolment.assistant.organisation.details.title = Your organisation details
 
-enrolment.firstName = First name
-enrolment.lastName = Last name
+label.mobilePhone = Mobile number
+optional.label = Optional
+enrolment.tradeName = <h1 class="govuk-heading-m">Trading name (Optional)</h1>
+enrolment.mobphone = <h1 class="govuk-heading-m">Mobile number</h1>
+enrolment.phone = <h1 class="govuk-heading-m">Telephone number</h1>
+enrolment.confirmedEmail = <h1 class="govuk-heading-m">Confirm email address</h1>
+enrolment.email = <h1 class="govuk-heading-m">Email address</h1>
+enrolment.firstName = <h1 class="govuk-heading-m">First name</h1>
+enrolment.lastName = <h1 class="govuk-heading-m">Last name</h1>
```

Comment:
why do we have html tags in here? They should be in the twirl template
(and the others...)

Reply:
Similar to the last ones i'm passing them like this: label = Label(content = HtmlContent(messages("enrolment.firstName"))),

@darrell-plant

Reply:
I've noticed some of our services use this method of having HTML within the Messages file - my last ticket I went through and removed the HTML which I was using. I think the Messages file should just be for plain text, although this does work and is done elsewhere, in my opinion we should keep our messages file strictly text only.